modules/k8s-nmstate-deploying-nmstate-CLI.adoc

```diff
@@ -92,6 +92,23 @@ metadata:
 EOF
 ----
 
+. Optional: If your cluster gets disconnected from a DNS server because of DNS connectivity issues, you can add the following DNS hostname configuration to the `NMState` CRD to build in health checks that can resolve these issues:
```

Comment:
Rather

>. Optional: If your cluster **has problems with DNS health check probe** because of DNS connectivity issues, you can add the following DNS hostname configuration to the `NMState` CRD to **customize the health check probe configuration**:

Reply:
:heavy_check_mark: